src/app/components/transaction-item/transaction-item.layout.tsx

```diff
@@ -32,19 +32,22 @@ export function TransactionItemLayout({
         flagImg={txIcon && txIcon}
         titleLeft={txTitle}
         captionLeft={
-          <HStack alignItems="center">
-            <Caption
-              overflow="hidden"
-              textOverflow="ellipsis"
-              maxWidth={{ base: '160px', md: 'unset' }}
-            >
-              {txCaption}
-            </Caption>
-            {txStatus && txStatus}
-          </HStack>
+          <VStack alignItems="start" gap="space.01">
+            <HStack alignItems="center">
+              <Caption
+                overflow="hidden"
+                textOverflow="ellipsis"
+                maxWidth={{ base: '160px', md: 'unset' }}
+              >
+                {txCaption}
+              </Caption>
+              {txStatus && txStatus}
+            </HStack>
+            {actionButtonGroupElement && actionButtonGroupElement}
+          </VStack>
         }
         titleRight={
-          rightElement ? rightElement : <styled.span textStyle="label.02">{txValue}</styled.span>
+          !actionButtonGroupElement && <styled.span textStyle="label.02">{txValue}</styled.span>
```

Comment:
Are you sure this will work? 

Before we set it to `rightElement` if defined but here it seems to just use a fallback if `!actionButtonGroupElement`

Reply:
> Are you sure this will work?
> 
> Before we set it to `rightElement` if defined but here it seems to just use a fallback if `!actionButtonGroupElement`

The position of the element was moved from being on right side to the bottom, so in this logic im checking if there is no `actionButtomGroupElement` then i should just show the `txValue`. When tested there were no issues. I am open to any suggestions for the layout or conditional rendering